### ISSUE TITLE
Can't Start HiddenEye-Reborn

### ISSUE THREAD

**Issue by Midget-Man** [D4rkS3c-Group/HiddenEyeReborn#86]
I do hope this isn't a crazy question but this is the first time coming back to a newer version of hiddeneye, one which doesn't have any steps on how to run it, since theres no file or directory that was downloaded to the path i was on, im unable to locate how to run it successfully. i did notice its located in .local/bin. and tried running the following ./hiddeneye-reborn, but i only get this without further instructions. any assistance is appreciated 

2024-08-31 14:47:44,658 [INFO] Success, you can access this via terminal!                                                                                                                         hiddeneye.py:16
2024-08-31 14:47:44,662 [INFO] THIS IS NOT PRODUCTION READY, STOP MAKING USELESS ISSUES PLEASE                                                                                                    hiddeneye.py:17
2024-08-31 14:47:44,664 [DEBUG] Logging level set to 10                                                                                                                                           hiddeneye.py:18
2024-08-31 14:47:44,682 [WARNING] Unable to verify connection with https://google.com due to <urlopen error [Errno -3] Temporary failure in name resolution>, returning False                  verification.py:26
2024-08-31 14:47:44,686 [DEBUG] Non-interactive mode set to False                                                                                                                                 hiddeneye.py:20

**Comment by sTiKyt** [comment]
@Midget-Man Try reading the output of line 16

**Comment by Midget-Man** [comment]
This is my result after reading line 16.
sudo ./hiddeneye-reborn
Traceback (most recent call last):
  File "/home/alpha/.local/bin/./hiddeneye-reborn", line 5, in <module>
    from hiddeneye_reborn.hiddeneye import execute
ModuleNotFoundError: No module named 'hiddeneye_reborn'

**Comment by sTiKyt** [comment]
@Midget-Man No, literally read the message you posted above, specifically the "THIS IS NOT PRODUCTION READY, STOP MAKING USELESS ISSUES PLEASE" part 😅
